libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/permissions/ProjectPermissions.java

```diff
@@ -113,7 +113,7 @@ public boolean isActionAllowed(RequestedAction action) {
             switch (action) {
                 case WRITE:
                 case ATTACHMENTS:
-                    return PermissionUtils.isUserAtLeast(UserGroup.CLEARING_ADMIN, user);
+                    return PermissionUtils.isClearingAdmin(user) && isUserInEquivalentToOwnerGroup() || PermissionUtils.isUserAtLeast(ADMIN, user);
```

Comment:
This is the same as https://github.com/sw360/sw360portal/pull/378/files#diff-cb022dd789f9102c57430a785e4f83c8R83. Maybe encapsulate as function with descriptive name